source/installation-guide/more-installation-alternatives/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -103,13 +102,13 @@ Installing Wazuh agent from sources
 
     .. code-block:: console
 
-     # $DIRECTORY/bin/wazuh-control stop 2> /dev/null
+     # $WAZUH_HOME/bin/wazuh-control stop 2> /dev/null
 
-    Remove files and service artifacts:
+    Remove installation folder:
```

Comment:
I would say `Remove the installation folder and all its content`

Reply:
Done!